### PR TITLE
Import Linux Kernel typos: letter V

### PR DESCRIPTION
**Pull request by Gelma**

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -12227,6 +12227,7 @@ vaguaries->vagaries
 vaiable->variable
 vaiables->variables
 vaiants->variants
+vaid->valid
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: also `void`

**Comment by luzpaz** [comment]
Once revision is made this commit LGTM

File: codespell_lib/data/dictionary.txt (hunk 1 above)
Comment (Gelma) [reply codespell_lib/data/dictionary.txt]: Thanks. Fixed.

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -12365,10 +12370,15 @@ villification->vilification
...
 vincinity->vicinity
+vinrator->vibrator
 violentce->violence
+virtaul->virtual
+virtiual->virtual
 virtualy->virtually
 virtuell->virtual
+virtural->virtual
 virture->virtue
+virual->virtual
```
Comment (luzpaz) [inline codespell_lib/data/dictionary.txt]: Can also be `viral`

**Pull request merged**